components/src/instrument/InstrumentInfo.tsx

```diff
@@ -1,77 +1,101 @@
   description: string
-  /** paired tiprack models */
-  tiprackModels?: string[]
-  /** if disabled, pipette & its info are grayed out */
-  isDisabled: boolean
   /** specs of mounted pipette */
   pipetteSpecs?: InstrumentDiagramProps['pipetteSpecs'] | null
-  /** classes to apply */
-  className?: string
-  /** classes to apply to the info group child */
-  infoClassName?: string
+  /** paired tiprack models */
+  tiprackModels?: string[]
   /** children to display under the info */
   children?: React.ReactNode
+  /** if true, show labels 'LEFT PIPETTE' / 'RIGHT PIPETTE' */
+  showMountLabel?: boolean | null
 }
 
 export function InstrumentInfo(props: InstrumentInfoProps): JSX.Element {
-  const has96Channel = props.pipetteSpecs?.channels === 96
+  const {
+    mount,
+    showMountLabel,
+    description,
+    tiprackModels,
+    pipetteSpecs,
+    children,
+  } = props
+
+  const has96Channel = pipetteSpecs?.channels === 96
   return (
     <Flex justifyContent={JUSTIFY_CENTER} gridGap={SPACING.spacing16}>
-      {props.mount === RIGHT && props.pipetteSpecs && (
+      {mount === RIGHT && pipetteSpecs ? (
         <InstrumentDiagram
-          pipetteSpecs={props.pipetteSpecs}
+          pipetteSpecs={pipetteSpecs}
           className={styles.pipette_icon}
-          mount={props.mount}
+          mount={mount}
         />
-      )}
+      ) : null}
+      {/* NOTE: the color is our legacy c-font-dark, which matches the other colors in this component **/}
+      <Flex
+        flexDirection={DIRECTION_COLUMN}
+        color="#4a4a4a"
+        gridGap={SPACING.spacing8}
+      >
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing4}>
+          <StyledText
+            as="h3"
+            fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+            textTransform={TYPOGRAPHY.textTransformCapitalize}
+          >
+            {showMountLabel && !has96Channel ? `${mount} pipette` : 'pipette'}
+          </StyledText>
+          <StyledText as="p" width="max-content" maxWidth="14rem">
+            {description}
+          </StyledText>
+        </Flex>
 
-      <Flex flexDirection={DIRECTION_COLUMN}>
-        <InfoItem
-          title={
-            props.showMountLabel && !has96Channel
-              ? `${props.mount} pipette`
-              : 'pipette'
-          }
-          value={props.description}
-        />
-        {props.tiprackModels != null
-          ? props.tiprackModels.map((model, index) => (
-              <InfoItem
-                key={`tiprackModel-${index}`}
-                title={index === 0 ? 'tip racks' : null}
-                value={model}
-              />
-            ))
-          : null}
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing4}>
+          <StyledText as="h3" fontWeight={TYPOGRAPHY.fontWeightSemiBold}>
+            Tip rack
+          </StyledText>
+          <ul>
+            {tiprackModels != null ? (
```

Comment:
from the props type, tiprackModels: string[] so checking its length would be better.

```suggestion
            {tiprackModels != null && tiprackModels.length >0 ? (
```